Source/AwsCommonRuntimeKit/auth/credentials/AwsCredentials.swift

```diff
@@ -37,13 +37,15 @@ public final class AwsCredentials {
         }
 
         guard let rawValue = (withByteCursorFromStrings(
-                accessKey,
-                secret,
-                sessionToken) { accessKeyCursor, secretCursor, sessionTokenCursor in
-            return aws_credentials_new(allocator.rawValue,
-                    accessKeyCursor,
-                    secretCursor,
-                    sessionTokenCursor, expirationTimeout)
+            accessKey,
+            secret,
+            sessionToken) { accessKeyCursor, secretCursor, sessionTokenCursor in
+            return aws_credentials_new(
+                allocator.rawValue,
+                accessKeyCursor,
+                secretCursor,
+                sessionTokenCursor,
+                expirationTimeout)
```

Comment:
I'm not personally a huge fan of this. The lines that get wrapped due to being too long blend in too much with the lines in the indented block of code:

```
            accessKey,
            secret,
            sessionToken) { accessKeyCursor, secretCursor, sessionTokenCursor in
            return aws_credentials_new(
                allocator.rawValue,
                accessKeyCursor,
                secretCursor,
                sessionTokenCursor,
                expirationTimeout)
```

this is due to whatever rules put "1-tab indentation of wrapped lines", and "{ open brace at end of line". We have this same setup in our C code and I hate it.

My first instinct would be to resolve this via 2-tab indentation for run-on lines?

but I also understand that there are a loooot of rules combining, and blending them all together with automation you occasionally get an ugly duckling like this. Skimming through the rest of the changes I didn't see this happening a ton. We could just "fix" this ugliness manually with a newline before the 1st line in the new block.

Your call which direction to take.

Reply:
I'd love to see this formatted like the following:
(on my todo list is to update our linter to enforce this style of formatting)

```swift
    sessionToken
) { accessKeyCursor, secretCursor, sessionTokenCursor in
    return aws_credentials_new(
                allocator.rawValue,
                accessKeyCursor,
                secretCursor,
                sessionTokenCursor,
                expirationTimeout
    )
```

Reply:
Thanks, I have fixed it manually for now.